server/app/views/applicant/TrustedIntermediaryDashboardView.java

```diff
@@ -130,6 +126,12 @@ public Content render(
     return layout.renderWithNav(request, personalInfo, messages, bundle, currentTisApplicantId);
   }
 
+  private ATag renderAddNewClientButton(Messages messages, Long tiGroupId) {
+    return new ATag()
+        .withHref(routes.TrustedIntermediaryController.showAddClientForm(tiGroupId).url())
+        .with(button(messages.at(MessageKey.BUTTON_ADD_NEW_CLIENT.getKeyName())));
+  }
```

Comment:
Question (non-blocking): Asking because I genuinely just don't know the answer: For buttons that redirect to other pages, do we prefer to put a `<button>` inside an `<a>`, or do we prefer to use `BaseHtmlView.asRedirectElement(button(), redirectUrl)`?

Reply:
I changed it!. Please let me know. Even I am not sure!

Reply:
This is going to be a USWDS button (using their button CSS classes), so we'll need it to be a button element.

Reply:
@melanie-exygy  - Do you have an example in the codebase?

Reply:
Changed it to BaseHtmlView.redirectButton(id,text,url)--> which returns a ButtonTag. Please let me know if it works.

Reply:
I thought that it needed to be a button to apply the USWDS button classes, but it doesn't.  Here's an example from the client cards:  [link styling example](https://github.com/civiform/civiform/blob/4902bcd1793739a9add7d59fc358aabb2c65976c/server/app/views/applicant/TrustedIntermediaryDashboardView.java#L445)

Reply:
Since in this case, the mocks are showing a blue button, you'd just need `.withClasses("usa-button")`.  I'd suggest using an ATag rather than a button because otherwise you'll have to fight against our default button styles to be able to apply the USWDS one.